solutions/hierarchy/tenant-env/folder-iam.yaml

```diff
@@ -1 +1,30 @@
-# TODO: PLACEHOLDER
\ No newline at end of file
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Grant GCP role Folder Viewer on Tenant's folder to Tenant's user group
+# AC-3(7) - ACCESS ENFORCEMENT | ROLE-BASED ACCESS CONTROL
+apiVersion: iam.cnrm.cloud.google.com/v1beta1
+kind: IAMPolicyMember
+metadata:
+  name: tenants.tenant1-tenant-folder-viewer-permissions # kpt-set: tenants.${tenant-name}-tenant-folder-viewer-permissions
+  namespace: hierarchy
+  annotations:
+    cnrm.cloud.google.com/ignore-clusterless: "true"
+spec:
+  resourceRef:
+    apiVersion: resourcemanager.cnrm.cloud.google.com/v1beta1
+    kind: Folder
+    name: tenants.tenant1 # kpt-set: tenants.${tenant-name}
+  role: roles/resourcemanager.folderViewer
```

Comment:
I'm also thinking of giving Essential Contacts admin and viewer roles (it would be 2 distinct setters) to the tenants

Reply:
spawned #271 to track this enhancement